assets/README.md

```diff
@@ -1,58 +1,154 @@
+@forward "00-base/colors";
+// Grab the fonts from this package for inclusion:
+@forward "00-base/fonts";
+```
+```
+// some-other-file.scss
+
+// This path depends on what's in your include paths...
+@use "path/to/example.scss" as *;
+// You can also scope it:
+@use "path/to/example.scss";
+.elementClass {
+  ... rules go here ...
+}
+```
+Creating a shared file is NOT required. You can also just ensure that you include the box-sizing styling:
+```
+// some-other-file.scss
+@use "00-base/mixins/box-sizing";
+.elementClass {
+  ... rules go here ...
+}
+```
+## Including SCSS Files
+To include a `.scss` file, please do NOT use `@import`. All previous uses of `@import` have been replaced by `@use` for [Sass Modules](https://sass-lang.com/documentation/at-rules/use) support:
+```
+// Scoped inclusion of variables. All variables will now
+// be under the variables scope.
+@use "00-base/variables";
+$my-new-var: variables.$column;
+// Or you can do this to place all variables
+// in the global scope for the file:
+@use "00-base/variables" as *;
+$my-new-var: $column;
+```
+## Variable Overrides
+All provided variables of this package allow for overrides by the user:
+```
+// You can define your overrides before the @use call.
+// Doing so will ensure the file you're calling with @use uses the overrides.
+$column: 60px;
+@use "00-base/variables";
+// You can also just pass the override to the file you're calling @use on:
+@use "00-base/variables" with (
+  $column: 60px;
+);
+```
+## Recommended Fonts
+Some reccomended fonts you can use can be found under `00-base/fonts`.
+## Example Compilation of Styles
+
+The following example uses gulp to compile all .scss files under src (with exceptions to those .scss files under `00-base/`). The script will place the compiled files under `dist/` within the same directories they originated.
+```
+// gulpfile.js
+const { src, dest, series } = require('gulp');
+const sass = require('gulp-sass');
+const rename = require('gulp-rename');
+const flatmap = require('gulp-flatmap');
+const del = require('del');
+const bourbon = require('bourbon');
+const path = require('path');
+// Replaces node_sass with sass.
+sass.compiler = require('sass');
+
+const compileSass = () => {
+  return src([
+    // Compile all scss files
+    './src/scss/**/*.scss',
+    // But skip over base directory files.
+    '!./src/scss/00-base/**'
+  ])
+  .pipe(rename(function(assetPath) {
+    // Sass will not compile files that start with an underscore,
+    // so rename them to remove it.
+    assetPath.basename = assetPath.basename.replace('_', '');
+    return assetPath;
+  }))
+  // Loops over all files in the stream with flatmap.
+  .pipe(flatmap((stream, file) => {
+    // Normally you should require/import the index file
+    // to get the includePaths used but that points to 
+    // the dist/ directory. Here we mimic that by just replacing
+    // dist/ with src/.
+    return stream
+    
+    .pipe(sass({
+      includePaths: [
```

Comment:
In this example, maybe we can use assets/index.js that you showed me?

Also, I don't think index.js is in the dist/folder, only fonts images and scss are being moved.